doc/ysh-faq.md

```diff
@@ -72,47 +72,56 @@ The rules are subtle, so we avoid inventing new ones.
 
 ## How do I write the equivalent of `echo -e` or `echo -n`?
 
-YSH removed these flags to `echo` because it has good alternatives
+YSH removed these flags to `echo` to solve their interference problems with
+strict backslash escaping and variable substitutions (see next).
 
-To escape variables, use the statically-parsed string language, rather than
-`echo -e`:
+To echo special characters denoted by escape sequences, choose a
+statically-parsed string literal, rather than `echo -e`:
 
-    echo $'tab \t newline \n'   # YES
-    echo j"tab \t newline \n"   # TODO: J8 notation
+    echo $'tab: \t' $indented       # YES ($IFS separated, shell compatible)
+    echo u'tab: \t' $indented       # J8 with \u{unicode} notation
+    echo b'tab: \t' $indented       # J8 with \u{unicode} and \y{byte} notation
+    echo $[u'tab: \t ' ++ indented] # string addition (not $IFS separated)
```

Comment:
I feel this section has gotten a bit complex

I do like the addition of the `++`, and double quotes are worth a mention

 I will merge this into another branch and then minimize / polish it a bit